submodules/wizard/views/step-appRestrictions.html

```diff
@@ -0,0 +1,86 @@
+								<input type="radio" name="accessLevel" id="access_level_full" class="radio-button input-hidden" value="full"{{#compare @root.data.accessLevel "===" "full"}} checked{{/compare}}/>
+								<label for="access_level_full" class="monster-button monster-button-fit">
+									{{appRestrictionsI18n.accessLevel.types.full}}
+								</label>
+								<input type="radio" name="accessLevel" id="access_level_restricted" class="radio-button input-hidden" value="restricted"{{#compare @root.data.accessLevel "!==" "full"}} checked{{/compare}}/>
+								<label for="access_level_restricted" class="monster-button monster-button-fit">
+									{{appRestrictionsI18n.accessLevel.types.restricted}}
+								</label>
+							</div>
+							<div>
+								<span class="help-tip help-tip-button" data-title="{{appRestrictionsI18n.accessLevel.help}}" data-toggle="tooltip">
+									{{telicon "question--circle" class="iconography-small iconography-help"}}
+								</span>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class="flex-item-2">
+					{{#monsterPanelText appRestrictionsI18n.accessLevel.important 'info'}}
+						<p>{{appRestrictionsI18n.accessLevel.notice}}</p>
+					{{/monsterPanelText}}
+				</div>
+			</div>
+			<div id="section_allowed_apps" {{#compare @root.data.accessLevel "===" "full"}}style="display: none"{{else}}class="visible"{{/compare}}>
+				<div class="section-title">
+					{{appRestrictionsI18n.apps.title}}
+				</div>
+				<div class="section-body">
+					<div class="flex-row-container">
+						<div class="flex-item-2">
+							{{appRestrictionsI18n.apps.description}}
+						</div>
+					</div>
+					<div class="flex-row-container app-list">
+					{{#each @root.apps}}
+						<div id="app_{{id}}" class="flex-item-3 flex-container app-item{{#ifInArray id @root.data.allowedAppIds}} visible"{{else}}" style="display:none"{{/ifInArray}} data-id="{{id}}">
+							<div class="wizard-card">
+								<div class="card-section title flex-container">
+									<div class="app-icon {{extraCssClass}}" style="{{#if icon}}background-image:url({{icon}}){{else}}background-color:#606069{{/if}}"></div>
+									<div class="app-name">
+										{{label}}
+									</div>
+									<label class="app-remove" href="javascript:void(0);">
+										{{telicon "trash-x" class="iconography"}}
+										<input class="app-selected input-hidden" type="checkbox" name="allowedAppIds[]" value="{{id}}"{{#ifInArray id @root.data.allowedAppIds}} checked{{/ifInArray}}/>
+									</label>
+								</div>
+								<div class="card-section description">
+									{{description}}
+								</div>
+							</div>
+						</div>
+					{{/each}}
+						<div class="flex-item-3 flex-container app-item app-add visible"{{#unless @root.anyAppToSelect}} style="display:none"{{/unless}}>
+							<div class="wizard-card">
+								<div class="card-section description flex-container">
+									<a class="monster-link blue" href="javascript:void(0);">
```

Comment:
The `href` attribute on `a` elements is not required, just be sure to use `Event.preventDefault` if a default behavior is already defined for the event you are binding to.

Reply:
:+1: 

In this specific case, though, I'll remove the `a` tag, because the click is handled by the container.